pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -241,6 +241,13 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		Name:     names.APPLIED_NAMESPACE,
 	})
 
+	// Add operator.openshift.io/v1/network to relatedObjects for must-gather
+	relatedObjects = append(relatedObjects, configv1.ObjectReference{
+		Group:    "operator.openshift.io",
+		Resource: "network",
```

Comment:
Oh. I just realized that the resource name is actually "networks" and not "network". @rcarrillocruz Can you please fix it?

Reply:
I created https://github.com/openshift/cluster-network-operator/pull/877